modules/openapi-generator-bazel/src/defs.bzl

```diff
@@ -0,0 +1,168 @@
+def openapi_generator_repositories(swagger_openapi_generator_cli_version = "4.0.3", swagger_openapi_generator_cli_sha1 = "4a7e6d7c82df64a1d869e68f28f23e6afd0f9d85", prefix = "io_bazel_rules_openapi_generator"):
```

Comment:
Should remove swagger references. Although OpenAPI 2.0 and Swagger 2.0 are one and the same, Swagger itself is a trademark of SmartBear and is in no relation to this project.

Reply:
Will do, thanks for the catch